src/main/resources/templates/fragments/reusable-components.html

```diff
@@ -1,43 +1,59 @@
-<div th:fragment="navbar">
-    <nav class="navbar navbar-dark bg-primary">
-        <div class="container-fluid">
-            <a class="navbar-brand">Hobby Collection</a>
-            <form class="d-flex">
-                <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search">
-                <button class="btn btn-outline-success" type="submit">Search</button>
-            </form>
-        </div>
-    </nav>
-</div>
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+
+<head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta charset="UTF-8">
+    <link rel="icon" href="https://www.shareicon.net/data/2017/01/18/872963_game_512x512.png">
+    <title>Hobby Collection: Reusable-components</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/js/bootstrap.bundle.min.js" integrity="sha384-b5kHyXgcpbZJO/tY9Ul7kGkf1S0CWuKcCD38l8YkeH8z8QjE0GmW1gYU5S9FOnJ0" crossorigin="anonymous"></script>
+</head>
```

Comment:
I added the title "Hobby Collection: Reusable-components" for the webpage, so the users can view what is the page for from the web tag.
Sources:
https://www.tutorialspoint.com/java/java_documentation.htm
https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3

Reply:
Since this is a fragment, the title is likely in the enclosing page, which would make this redundant.